python/ray/util/tracing/tracing_helper.py

```diff
@@ -244,7 +244,7 @@ def _actor_hydrate_span_args(class_: _nameable, method: _nameable):
         "ray.function": f"{class_}.{method}",
         "ray.pid": str(os.getpid()),
         "ray.job_id": runtime_context["job_id"].hex(),
```

Comment:
Actually, can we also do it for job_id (or any other Cython ids here)?

Reply:
Yes, I do plan to do the same thing for other ids but in follow-up PRs. Does that work for you?

Reply:
Sure